recipes/conf.rb

```diff
@@ -0,0 +1,22 @@
+#
+# Cookbook:: auditd
+# Recipe:: default
+#
+# Copyright:: 2018, Chef Software, Inc.
```

Comment:
Is this intentional?

Reply:
I also agree that we should not add this here

Reply:
If we're shipping it we should add our copyright so this is entirely appropriate